freqtrade/analyze.py

```diff
@@ -301,15 +301,20 @@ def get_signal(pair: str, signal: SignalType) -> bool:
         return False
 
     if dataframe.empty:
+        logger.warning('Empty dataframe for pair %s', pair)
         return False
 
     latest = dataframe.iloc[-1]
 
     # Check if dataframe is out of date
     signal_date = arrow.get(latest['date'])
     if signal_date < arrow.now() - timedelta(minutes=10):
+        logger.warning('Too old dataframe for pair %s', pair)
         return False
 
+    # FIX: 20180109, there could be some confusion because we will make a
```

Comment:
I'm not sure if I understand that correctly. `latest[signal.value] == 1` should yield in a bool. So it should be a already a bool in every case or not?